lib/src/widgets/slider_title.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+
+class SliderTitle extends StatelessWidget {
+  const SliderTitle(
+    this.title,
+    this.text, {
+    Key? key,
+  }) : super(key: key);
+
+  final String title;
+  final String text;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(top: 8.0),
+      child: Row(
+        children: <Widget>[
+          Container(
+            alignment: Alignment.bottomCenter,
+            margin: const EdgeInsets.only(left: 10.0),
+            child: Opacity(
+              opacity: 0.7,
+              child: Text(
+                title,
+                style: Theme.of(context).textTheme.headline6?.copyWith(
+                      fontSize: 18,
+                    ),
+              ),
+            ),
+          ),
+          const Spacer(),
+          Container(
+            alignment: Alignment.bottomCenter,
+            margin: const EdgeInsets.only(right: 10.0),
+            child: Text(
+              text,
+              style: Theme.of(context).textTheme.headline5?.copyWith(
+                    fontSize: 18,
+                  ),
+            ),
+          ),
+        ],
+      ),
+    );
```

Comment:
Let me know if you think it's better or not.

```dart
return Container(
  alignment: Alignment.bottomCenter,
  padding: EdgeInsets.symmetric(horizontal: 10).copyWith(top: 8),
  child: Row(
    mainAxisAlignment: MainAxisAlignment.spaceBetween,
    children: <Widget>[...],
  ),
);
```

Reply:
I really like your idea!
Changed it to EdgeInsets.fromLTRB to make it const, and used margin instead of padding.
Please let me know what you think :)

Reply:
You can use const on symmetric actually, but that's okay for now. I do have some packages that have requirement for RTL, so I use neither left nor right as possible.